presto-release-tools/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesTask.java

```diff
@@ -142,20 +144,26 @@ public void run()
                 .collect(toImmutableMap(identity(), this::extractReleaseNotes));
 
         log.info("Collecting author information");
-        Map<String, String> userByLogin = new HashMap<>();
+        Map<String, String> authorByLogin = new HashMap<>();
```

Comment:
Lets add the dedup logic as discussed

Reply:
Used a single login to name map, to dedupe based on login